ciao-deploy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM clearlinux
+MAINTAINER marcos.simental.magana@intel.com
+
+ARG swupd_args
+ENV HOME=/root/
+
+RUN swupd bundle-add sysadmin-hostmgmt go-basic os-core-dev kvm-host os-common $swupd_args
```

Comment:
os-core-dev is a bug bundle that provides gcc which is needed to compile some of the ciao dependencies.

you can replace this bundle with c-basic which also provides gcc in a smaller bundle.